app/pages/system/UtilizationPage.tsx

```diff
@@ -55,67 +52,70 @@ SystemUtilizationPage.loader = async () => {
       query: capacityQueryParams,
     }),
     apiQueryClient.prefetchQuery('sledList', {}),
+    apiQueryClient.prefetchQuery('utilizationView', {}),
+    apiQueryClient.prefetchQuery('siloUtilizationList', {}),
   ])
   return null
 }
 
 export function SystemUtilizationPage() {
-  const { data: sleds } = usePrefetchedApiQuery('sledList', {})
-  const { data: silos } = usePrefetchedApiQuery('siloList', {})
+  const { data: siloUtilizationList } = usePrefetchedApiQuery('siloUtilizationList', {})
 
-  const capacity = totalCapacity(sleds.items)
+  const { totalAllocated, totalProvisioned } = totalUtilization(siloUtilizationList.items)
 
   return (
     <>
       <PageHeader>
         <PageTitle icon={<Metrics24Icon />}>Utilization</PageTitle>
       </PageHeader>
 
+      <h2 className="flex items-center gap-1.5 p-3 text-mono-sm text-secondary">
+        Capacity available
+      </h2>
       <div className="mb-12 flex min-w-min flex-col gap-3 lg+:flex-row">
-        <CapacityMetric
-          icon={<Ssd16Icon />}
-          title="Disk utilization"
-          metricName="virtual_disk_space_provisioned"
-          valueTransform={bytesToTiB}
-          capacity={capacity.disk_tib}
-        />
-        <CapacityMetric
+        <CapacityBar
           icon={<Cpu16Icon />}
-          title="CPU utilization"
-          metricName="cpus_provisioned"
-          capacity={capacity.cpu}
+          title="CPU"
+          unit="nCPUs"
+          provisioned={totalProvisioned.cpus}
+          quota={totalAllocated.cpus}
+        />
+        <CapacityBar
+          icon={<Ssd16Icon />}
+          title="Disk"
+          unit="TiB"
+          provisioned={bytesToTiB(totalProvisioned.storage)}
+          quota={bytesToTiB(totalAllocated.storage)}
         />
```

Comment:
It's worth swapping this around to be after memory. CPU, Memory are kind of like PB&J, they just go together.